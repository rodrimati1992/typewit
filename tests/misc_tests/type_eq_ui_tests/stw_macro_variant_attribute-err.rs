typewit::simple_type_witness!{
    enum TypeParams {
        #[asdkaspokaopskd]
        U8 = u8,
        #[cfg(any())]
        U16 = u16 
    }
}
