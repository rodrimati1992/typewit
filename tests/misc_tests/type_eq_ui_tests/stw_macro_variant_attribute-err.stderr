error: cannot find attribute `asdkaspokaopskd` in this scope
 --> tests/misc_tests/type_eq_ui_tests/stw_macro_variant_attribute-err.rs:3:11
  |
3 |         #[asdkaspokaopskd]
  |           ^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/misc_tests/type_eq_ui_tests/stw_macro_variant_attribute-err.rs:8:2
  |
8 | }
  |  ^ consider adding a `main` function to `$DIR/tests/misc_tests/type_eq_ui_tests/stw_macro_variant_attribute-err.rs`

error[E0599]: no variant or associated item named `U16` found for enum `TypeParams` in the current scope
 --> tests/misc_tests/type_eq_ui_tests/stw_macro_variant_attribute-err.rs:6:9
  |
1 | / typewit::simple_type_witness!{
2 | |     enum TypeParams {
3 | |         #[asdkaspokaopskd]
4 | |         U8 = u8,
5 | |         #[cfg(any())]
6 | |         U16 = u16
  | |         ^^^ variant or associated item not found in `TypeParams<u16>`
7 | |     }
8 | | }
  | |_- variant or associated item `U16` not found for this enum
